README.rst

```diff
@@ -1,148 +1,262 @@
+    {"fiware-facts":"Up and running..."}
 
-- Fixing some magic numbers. [Guillermo Jimenez Prieto]
 
-- Fixing coverage. [Guillermo Jimenez Prieto]
+Please refer to the `Installation and administration guide
+<https://github.com/telefonicaid/fiware-cloto/tree/develop/doc/admin_guide.rst#end-to-end-testing>`_ for details.
 
-- Improving signal stability checking each fact with the previous one.
-  [Guillermo Jimenez Prieto]
+Acceptance tests
+----------------
 
-- Fixing a multitenacy bug. [Guillermo Jimenez Prieto]
+Fiware-facts acceptance tests are included into fiware-cloto repository (https://github.com/telefonicaid/fiware-cloto).
 
-- Fixing a multitenacy bug. [Guillermo Jimenez Prieto]
+ Requirements
 
-- Repaired multitenacy bug with serverid in lists. [Guillermo Jimenez
-  Prieto]
+  - Python 2.7 or newer
+  - pip installed (http://docs.python-guide.org/en/latest/starting/install/linux/)
+  - virtualenv installed (pip install virtalenv)
+  - Git installed (yum install git-core / apt-get install git)
 
-v1.1.0 (2015-03-04)
--------------------
+ Environment preparation:
 
-New
-~~~
+  - Create a virtual environment somewhere, e.g. in ENV (virtualenv ENV)
+  - Activate the virtual environment (source ENV/bin/activate)
+  - Change to the test/acceptance folder of the project
+  - Install the requirements for the acceptance tests in the virtual environment (pip install -r requirements.txt --allow-all-external).
+  - Configure file in fiware-cloto/tests/acceptance_tests/commons/configuration.py adding the keystone url, and a valid, user, password and tenant ID.
 
-- New: dev: preparing release @release. [Guillermo Jimenez Prieto]
+ Tests execution
 
-v1.0.0 (2015-02-24)
--------------------
+  Change to the fiware-cloto/tests/acceptance_tests folder of the project if not already on it and execute:
+  ::
 
-New
-~~~
+        lettuce_tools -ft features/context_update.feature --tags=skip
 
-- Building travis. [Guillermo Jimenez Prieto]
 
-Fix
-~~~
+In the following document you will find the steps to execute automated
+tests for the Policy Manager GE:
 
-- Fixing an acceptance test and cobertura. [Guillermo Jimenez Prieto]
+- `Policy Manager acceptance tests <https://github.com/telefonicaid/fiware-cloto/tree/develop/cloto/tests/acceptance_tests/README.md>`_
 
-- Updating unittests and adding new data to build.sh in order to build
-  in jenkins. [Guillermo Jimenez Prieto]
 
-- Fixing multitenacy bug and gunicorn deployment. [Guillermo Jimenez
-  Prieto]
+Advanced topics
+===============
 
+- `Installation and administration <https://github.com/telefonicaid/fiware-cloto/tree/develop/doc/admin_guide.rst>`_
+- `User and programmers guide <https://github.com/telefonicaid/fiware-cloto/doc/tree/develop/doc/user_guide.rst>`_
+- `Open RESTful API Specification <https://github.com/telefonicaid/fiware-cloto/tree/develop/doc/open_spec.rst>`_
+- `Architecture Description <https://github.com/telefonicaid/fiware-cloto/tree/develop/doc/architecture.rst>`_
 
```

Comment:
Add the section *Support* before *License* with the content:

Support
======

Ask your thorough programming questions using stackoverflow and your general questions on FIWARE Q&A. In both cases please use the tag fiware-cloto

Top